package.json

```diff
@@ -25,7 +25,8 @@
     "src/**/*.js": "eslint --max-warnings=0"
   },
   "dependencies": {
-    "classnames": "^2.2.5"
+    "classnames": "^2.2.5",
+    "npm": "^5.8.0"
```

Comment:
I believe this might have happened when I noticed my NPM was out of date and instead of updating globally, I updated locally.
I think a note saying 'NPM is required' would do just fine.
Depending on @AaronCoplan's plan, this should be an easy fix.